src/components/Header/AccountMenu/index.tsx

```diff
@@ -0,0 +1,26 @@
+import AvatarPlaceholder from "@/assets/Images/Avatar.jpg";
+import { Menu } from "lucide-react";
+import Image from "next/image";
+import styles from "./account-menu.module.css";
+import { StaticImport } from "next/dist/shared/lib/get-img-props";
+
+type Props = {
+  username?: string;
+  avatar?: StaticImport | string
+}
+
+const usernamePlaceholder = "Kadoodle"
+
+const AccountMenu: React.FC<Props> = ({ username = usernamePlaceholder, avatar = AvatarPlaceholder }) => {
```

Comment:
eu removeria esses campos default e passaria eles na chamada desse componente

Reply:
coloquei esses valores default pra mockar o username e o avatar e depois remover quando for feita a integração com o serviço, mas realmente não faz muito sentido adicionar algo pra remover depois. vou fazer essa correção.